packages/logs/src/domain/configuration.spec.ts

```diff
@@ -22,4 +23,72 @@ describe('validateAndBuildLogsConfiguration', () => {
       ).toBeTrue()
     })
   })
+
+  describe('forwardConsoleLogs if ff forward-logs enabled', () => {
+    let displaySpy: jasmine.Spy<typeof display.error>
+    const errorMessage =
+      'Forward Console Logs should be "all" or an array with allowed values "log", "debug", "info", "warn", "error"'
+
+    beforeEach(() => {
+      displaySpy = spyOn(display, 'error')
+      updateExperimentalFeatures(['forward-logs'])
+    })
+
+    afterEach(() => {
+      resetExperimentalFeatures()
+    })
+
+    it('does not validate the configuration if an incorrect string is provided', () => {
+      validateAndBuildLogsConfiguration({ ...DEFAULT_INIT_CONFIGURATION, forwardConsoleLogs: 'foo' as any })
+
+      expect(displaySpy).toHaveBeenCalledOnceWith(errorMessage)
+    })
+
+    it('does not validate the configuration if an incorrect api is provided', () => {
+      validateAndBuildLogsConfiguration({ ...DEFAULT_INIT_CONFIGURATION, forwardConsoleLogs: ['dir'] as any })
+
+      expect(displaySpy).toHaveBeenCalledOnceWith(errorMessage)
+    })
+
+    it('defaults to an empty array', () => {
+      expect(validateAndBuildLogsConfiguration(DEFAULT_INIT_CONFIGURATION)!.forwardConsoleLogs).toEqual([])
+    })
+
+    it('is set to provided value', () => {
+      expect(
+        validateAndBuildLogsConfiguration({ ...DEFAULT_INIT_CONFIGURATION, forwardConsoleLogs: ['log'] })!
+          .forwardConsoleLogs
+      ).toEqual(['log'])
+    })
+
+    it('contains "error" when forwardErrorsToLogs is enabled', () => {
+      expect(
+        validateAndBuildLogsConfiguration({ ...DEFAULT_INIT_CONFIGURATION, forwardErrorsToLogs: true })!
+          .forwardConsoleLogs
+      ).toEqual(['error'])
+    })
+
+    it('contains all apis when forwardConsoleLogs is set to "all"', () => {
+      expect(
+        validateAndBuildLogsConfiguration({ ...DEFAULT_INIT_CONFIGURATION, forwardConsoleLogs: 'all' })!
+          .forwardConsoleLogs
+      ).toEqual(Object.keys(ConsoleApiName) as ConsoleApiName[])
+    })
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** it could be nice to have a test where both options are enabled, to ensure that `error` is not tracked twice (`removeDuplicates`)‏

Reply:
Sure.